contrib/cf/pcf-tile/Makefile

```diff
@@ -0,0 +1,38 @@
+SHELL = /bin/bash
+
+GENERATE_TILE = docker run --rm -v ${CURDIR}:/workspace -w /workspace --name=tile-generator cfplatformeng/tile-generator tile build
```

Comment:
The image name `cfplatformeng/tile-generator` is implicitly interpreted as `cfplatformeng/tile-generator:latest`, where `latest` denotes a mutable tag. (i.e. Over time, this will point to different, newer images.) This impairs the determinism and repeatability of the tasks that use this.

Please update this to reference a specific version of the image, i.e. referenced using a semver tag.

Reply:
@krancour thanks! Fixed it with the same version as pull.